rnd/nextgenautogpt/nextgenautogpt/__main__.py

```diff
@@ -1,39 +0,0 @@
-import multiprocessing as mp
```

Comment:
Why are you deleting what I´ve done?

Reply:
Renaming + updating the code to adapt fastapi, end up shown as delete and new file.